packages-ts/gauntlet-terra-contracts/src/commands/contracts/multisig/updateAdmin.ts

```diff
@@ -0,0 +1,45 @@
+import { CATEGORIES } from '../../../lib/constants'
```

Comment:
Can we please move this command to a `../cw4_group/...` subfolder, as this is the canonical name of the contract.

Also, why is this in the `gauntlet-terra-contracts` package and not in the `gauntlet-terra-cw-plus` package, which is the name (`cw-plus`) of the vendor source of contracts? cc @RodrigoAD

If there is a blocker that we can't do it now, let's at least be explicit in the folder structure and inline docs that this should be moved to another location.

Reply:
This is blocked until we move abstract commands from `gauntlet-terra-contracts` to `gauntlet-terra`. Opening issue

Reply:
https://github.com/smartcontractkit/chainlink-terra/issues/136